scripts.js

```diff
@@ -1,24 +1,64 @@
-/* For picking letters from alphabet keypad. Returns the
-   letter chosen or "already picked"
-*/
-function pickLetter(letter) {
-  "use strict";
-  var result;
-
-  if (letter.className === "letter-button") {
-    result = letter.innerHTML;
-    letter.className = "letter-disabled";
-  } else {
-    result = "already picked";
-  }
-
-  console.log(result);    // for testing, remove later
-  return result;
+/*
+ * Pick from alphabet keypad. Returns the letter chosen.
+ */
+$("#alphabet-keypad").on("click", ".letter-button", pickLetter);
+
+function pickLetter() {
+  var letterChosen = $(this);
+  
+  letterChosen
+    .removeClass("letter-button")
+    .addClass("letter-disabled");
+
+  // Next two lines to be refactored into interface for
+  //   checking if the letter is in the word.
+  
+  letterChosen = letterChosen.html();
+  
+  verify(letterChosen);
 }
 
-// adding dictionary and word filter //
+// placeholder for verifying the letter
+function verify(letter) {
+  console.log(letter);
+}
 
 
+/*
+ * Hangman Object with methods loseLife() and reset()
+ */
+var hangmanObject = function () {
+  var livesLost = 0,
+      maxLife = 7;
+  
+  return {
+    loseLife : function () {
```

Comment:
@simonxca I've assigned [that Trello card](https://trello.com/c/iIOLQECG/12-when-i-make-a-bad-guess-i-want-to-see-a-new-body-part-appear-and-the-letter-to-show-in-the-graveyard-so-i-have-consequences-for-#) to myself a while back wherein any wrong guesses would update the body parts accordingly. I've started work on the same. I've been going through your changes and that of @piecedigital . I have created a branch from your branch and plan to add some of @piecedigital  changes on top of it so that I could reuse what's already done by you both. Thank you and @piecedigital for your changes. Hope you're okay with this plan.